README.md

```diff
@@ -85,6 +85,39 @@ module.exports = {
 }
 ```
 
+To customize compression, you can add optional parameters to the zopfli library: ([see here for details on various options)](https://github.com/pierreinglebert/node-zopfli#options)
+
+```javascript
+module.exports = {
+  plugins: [
+    {
+      resolve: 'gatsby-plugin-zopfli',
+      zopfli: {
+        verbose: false,
+        verbose_more: false,
+        numiterations: 15,
+        blocksplitting: true,
+        blocksplittinglast: false,
+        blocksplittingmax: 15
+      }
```

Comment:
Could you move the new zopfli settings to the options-object, so all settings will be in the same place?
```
  resolve: 'gatsby-plugin-zopfli',
      options: {
        verbose: false,
        verbose_more: false,
        numiterations: 15,
        extensions: [],
        path: '',
        ...
```

Reply:
Thanks, I actually documented incorrectly, it will now be in options->compression (my reasoning to have it as a sub-property is so that extraneous options are not fed to the underlying zopfli library)